modules/machine-health-checks-about.adoc

```diff
@@ -5,26 +5,30 @@
 [id="machine-health-checks-about_{context}"]
 = About MachineHealthChecks
 
-MachineHealthChecks automatically repairs unhealthy Machines in a particular
-MachinePool.
+A MachineHealthCheck is a resource which allows users to define conditions under
+which Machine's within a Cluster should be considered unhealthy and should be
+automatically remediated.
 
-To monitor machine health, you create a resource to define the
-configuration for a controller. You set a condition to check for, such as
-staying in the `NotReady` status for 15 minutes or displaying a permanent condition
-in the node-problem-detector, and a label for the set of machines to monitor.
+When defining a MachineHealthCheck, users specify conditions to check for and a
+timeout for each of the conditions.
+If any of these conditions is met on the Machine's Node for the duration of the timeout,
+the Machine will be remediated.
+The action of remediating a Machine will trigger a new Machine to be created,
+to replace the failed one.
+
+Fo example, you mat set a condition such as staying in the `NotReady` state
+for 15 minutes or displaying a permanent condition in the node-problem-detector.
 
 [NOTE]
 ====
-You cannot apply a MachineHealthCheck to a machine with the master role.
+Please note that MachineHealthChecks currently *only* support Machines that are
```

Comment:
since this appears in a section named "Note", i think we should drop the "Please note that"

Reply:
![Screenshot 2020-05-13 at 14 36 12](https://user-images.githubusercontent.com/9232216/81819397-2d084000-9527-11ea-8f6b-3bedd5c2cf7f.png)
It looks like this when rendered, no "Note" anywhere to be seen, WDYT?

Reply:
ok, yeah. i'm cool with that, it looked weird when i rendered earlier.